src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] according to the requirements The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).